docs/user/ovirt/customization.md

```diff
@@ -0,0 +1,45 @@
+# Customizing a template during installation
```

Comment:
This entire document is very confusing to me.

It mixes the during, before installing a lot. It's not clear to me what you can do at each.

Also the during is too confusing, I have no clue enable when do you want the user to make the change.

Also at one point this document says stop the installer when the template is created, modify it and then restart with and env..
^^ all this is a really bad ux. No document should recommend this type of high-touch-time based workflow. Please fix it.

Also, this document points to code for options available for configuration.. that's not acceptable. Please clearly document so the customization.

Lastly this document doesn't look anything like docs/aws/customization of any other cloud ones. Please make sure the documents are consistent. It's important for readers to correctly anticipate.